Readme fix (#81)
ClarkSource/k8t: README.md

* `get_secret(key: str)` - provides a secret value from a given provider (see [here](#managing-secrets))
* `bool(value: Any)` - casts value to boolean ("true", "on", "yes", "1", 1 are considered as `True`)

> ## Overriding flow
> 
> Overriding flow is from low score (1) to high (4) means more-specific configuration will be on top.
> 
> So variables and templates will be ovveriden by flow `project` -> `environments` -> `clusters` -> `cluster-specific-environments`

Reviewer: ```suggestion
## Configuration inheritance

Configuration, values and templates are used according to the scope they are in. The following snippet shows an example project with low scores (1) and high scores (4) for evaluation order.

So variables and templates will be overridden from `project` -> `environments` -> `clusters` -> `cluster-environments` resulting in more specific configuration overriding lower values.
```

maybe sth like this?

> ```.                                        (1) # k8t new project .

Reviewer: love this :+1:

> Create a new environment

Reviewer: ```suggestion
Create a new global environment
```

```bash

> $ k8t new environment staging -c MyCluster

Reviewer: actually both are possible, the idea being that you could globally define something for the staging environment as well as for the staging environment in a particular cluster

Author: TBH I feel we need kinda flow here

```

> 

Reviewer: ```suggestion

And a new cluster environment

```bash
k8t new environment staging -c MyCluster
```

```

$ k8t new cluster MyCluster
```

Create a new global environment

```bash

> $ k8t new environment staging -c MyCluster

Reviewer: ```suggestion
$ k8t new environment staging
```

```

And a new cluster environment

```bash
k8t new environment staging -c MyCluster
Generate a new deployment template for cluster MyCluster (for a list of available templates see the `k8t new template --help`)

> 

Reviewer: can't really comment this with a suggestion, but the inline code snippets are broken
